rewritev/Main.hs

```diff
@@ -70,6 +85,23 @@ runWithArgs as = do
 
   proj <- guessProj src
 
+  cabal <- findCabal proj
+  let cab = case cabal of
+              Just c -> proj </> c
+              Nothing -> error "No Cabal"
+  gpd <- readGenericPackageDescription silent cab
+  let cn = case condLibrary gpd of
+             Just c@(CondNode _ _ _) -> c
+             Nothing -> error "No Library"
+  let libs = foldr (\l acc -> l:acc) [] cn
+      modules = concat $ map exposedModules libs
+      sources = concat $ map (hsSourceDirs . libBuildInfo) libs
+      others = concat $ map (otherModules . libBuildInfo) libs
+      paths = sources ++ (map MN.toFilePath $
+              (exposedModules $ condTreeData cn) ++ modules ++ others ++
+              (otherModules $ libBuildInfo $ condTreeData cn))
+      proj' = map (proj </>) paths
+
```

Comment:
This seems like it could be useful for applications other than RewriteV/Nebula- maybe we should move it to G2.Translation.Cabal.Cabal?